clean_md_commodities.py

```diff
@@ -22,287 +22,298 @@
-df.replace(to_replace='PRECIOUS METALS (?)', value= 'PRECIOUS METALS')
-df.replace(to_replace='PRECIOUS-METALS (?)', value= 'PRECIOUS METALS')
-df.replace(to_replace='PRECIOUS-METAL (?)', value= 'PRECIOUS METALS')
-df.replace(to_replace='QUARTZ (?)', value= 'QUARTZ')
-df.replace(to_replace='RUNGSTEN', value='TUNGSTEN')
-df.replace(to_replace='SHEELITE', value='SCHEELITE')
-df.replace(to_replace='SIILVER', value='SILVER')
-df.replace(to_replace='SILCA SAND', value='SILICA SAND')
-df.replace(to_replace='SILVE', value='SILVER')
-df.replace(to_replace='SILVER GOLD (?)' , value='SILVER, GOLD')
-df.replace(to_replace='SILVER HALOID SALTS', value='SILVER HALIDE SALTS')
-df.replace(to_replace='PRECIUOS-METALS', value='PRECIOUS METALS')
-df.replace(to_replace= 'SILVER)', value='SILVER')
-df.replace(to_replace='SILVER-SINC-LEAD', value='SILVER, ZINC, LEAD')
-df.replace(to_replace='SIVER', value='SILVER')
-df.replace(to_replace='SODIUM SUFATE', value='SODIUM SULFATE')
-df.replace(to_replace='SODUIM NITRATE', value='SODIUM NITRATE')
-df.replace(to_replace='STIVNITE', value='STIBNITE')
-df.replace(to_replace='SULFIDES (?)', value='SULFIDES')
-df.replace(to_replace='SULPHUR DEPOSITES', value='SULPHUR DEPOSITS') 
-df.replace(to_replace='SULVIDES AND PRECIOUS METALS', value='SULFIDES, PRECIOUS METALS')
-df.replace(to_replace='TELLINIUM', value= 'TELLURIUM')
-df.replace(to_replace='TITANIUM(?)', value='TITANIUM')
-df.replace(to_replace='TRAVERTINE (BUILDING STONE?)', value='TRAVERTINE')
-df.replace(to_replace='TUGNSTEN', value='TUNGSTEN')
-df.replace(to_replace='TUGSTEN', value='TUNGSTEN')
-df.replace(to_replace='TUNGITEN', value='TUNGSTEN')
-df.replace(to_replace='TUNGSTEN (?)', value='TUNGSTEN')
-df.replace(to_replace='TUNGSTON', value='TUNGSTEN')
-df.replace(to_replace='TUNSGTEN', value='TUNGSTEN')
-df.replace(to_replace='TUNSTEN', value='TUNGSTEN')
-df.replace(to_replace='TURQUOIS', value='TURQUOISE')
-df.replace(to_replace='URANIUM (?)', value='URANIUM')
-df.replace(to_replace= 'VANADIUM (?)', value= 'VANADIUM')
-df.replace(to_replace='VANADIUML ZINC', value= 'VANADIUM ZINC')
-df.replace(to_replace='VANANDINITE', value= 'VANADINITE')
-df.replace(to_replace='VANUDIUM', value= 'VANADIUM')
-df.replace(to_replace='VARICITE', value= 'VARISCITE')
-df.replace(to_replace='VOLCANID ASH', value= 'VOLCANIC ASH')
-df.replace(to_replace='WALLASTONITE', value='WOLLASTONITE')
-df.replace(to_replace='WOLASTONITE', value='WOLLASTONITE')
-df.replace(to_replace='WOLFENITE', value='WULFENITE')
-df.replace(to_replace='WOLLASTORITE', value='WOLLASTONITE')
-df.replace(to_replace='WULPHUR', value= 'SULPHUR')
-df.replace(to_replace='ZEOLITE (?)', value='ZEOLITE')
-df.replace(to_replace='ZEOLITES (?)', value='ZEOLITES')
-df.replace(to_replace='ZINC (?)', value='ZINC')
-df.replace(to_replace='ZINC?', value='ZINC')
-df.replace(to_replace='ZINCE', value='ZINC')
-df.replace(to_replace='ZINE', value='ZINC')
-df.replace(to_replace='ZINE (?)', value='ZINC')
 

```

Comment:
In the end, you should take your modified spreadsheets and compare them again to make sure that all commodities match up. So, basically just re-running the intersection and not_in code to make sure that all commodities in the md_files spreadsheet are now accounted for in the master commodities spreadsheet.